docs/source/API/core/KokkosConcepts.rst

```diff
@@ -102,8 +102,7 @@ and ``OpenMPTarget``, the current state of the Kokkos |ExecutionSpaceTwo|_ conce
 
     template <typename Ex>
     concept ExecutionSpace =
-        CopyConstructible<Ex> &&
-        DefaultConstructible<Ex> &&
+        SemiRegular<Ex> &&
```

Comment:
https://github.com/kokkos/kokkos/pull/5398 (Kokkos 4.0.00) introduced the comparison operator. I'm not sure if that's worth mentioning.